src/lib/getKeyDownHandlers.ts

```diff
@@ -34,9 +42,20 @@ const getKeyDownHandlers = (
     keyHandlers[componentPart] = {
       onKeyDown: (event: KeyboardEvent) => {
         handledActions.forEach(actionName => {
+          let keyCombinations = componentPartKeyAction[actionName].keyCombinations
+
+          if (isRtl) {
+            keyCombinations = keyCombinations.map(keyCombination => {
+              if (rtlKeyMap[keyCombination.keyCode]) {
+                keyCombination.keyCode = rtlKeyMap[keyCombination.keyCode]
+              }
+              return keyCombination
+            })
+          }
+
```

Comment:
We can move this code to a separate function and test it separately:
```js
const keyCombinations = normalizeKeyCombinations(componentPartKeyAction[actionName], isRtl)
```

Reply:
We can :) But I think this part of the code is very trivial to move it to separate function and test it because it's mainly key mapping.
I tested that correct action handlers are invoked based on key pressed, and this seems to be an important part.
Also, creating one more function leads to having one more `rtl` prop dependency, hence `getKeyDownHandlers` will do nothing about this param but just passing it further.